### ISSUE TITLE
Development for V3 refresher

### ISSUE THREAD

**Issue by drobny** [lwe/simple_enum#140]
I have been talking with @lwe about the future of this gem, I am sure we all agree he has done a fantastic job maintaining and developing this library over the years but he is now not working with Ruby at all and would be great if some of us could step up and maybe help out with some of the workload.

Maybe we could (re)discuss what people would like to see in v3 of simple_enum? I know the initial decisions were made a couple of years ago and may / may not be relevant anymore (based on the improvements to the AR enum functionality in the last couple of Rails versions) or there might be some further ideas people would like integrated.

Basically, I am hoping this will generate some sort of up-to-date discussion about what path people would like simple_enum to go down in the future, whether that be feature enhancements, maintenance or just general observations then that would be great.
